src/CONST.ts

```diff
@@ -1640,6 +1640,7 @@ const CONST = {
     FORM_CHARACTER_LIMIT: 50,
     LEGAL_NAMES_CHARACTER_LIMIT: 150,
     LOGIN_CHARACTER_LIMIT: 254,
+    CATEGORY_NAME_LIMIT: 256,
```

Comment:
Unaware about the limits on each feature, hence a question, do we document the limit somewhere?  If you see L1645, we have a generic `TITLE_CHARACTER_LIMIT: 100`, which I thought would be used here?

Reply:
The API validates these differently. I don't think there's documentation other than the CONSTS defined in the API. I checked this value against the API and 256 is correct for category names.